README.md

```diff
@@ -111,8 +114,23 @@ Set `$GOPATH` variable to point to your Go workspace directory and add `$GOPATH/
   - `genfiles/server.crt`
   - `genfile/p256-pubkey.pem`
 
-6. Run the server and the signer using
+6. Run the trillian-map server 
 
   ```sh
-  goreman start
+docker-compose up trillian-map -d
```

Comment:
` docker-compose up trillian-map -d` fails with `ERROR: No such service: -d` has to be `docker-compose up -d trillian-map` instead.